udata_front/theme/gouvfr/assets/js/api/datasets.js

```diff
@@ -73,5 +73,5 @@ export function getFrequenciesUrl () {
  * @returns {Promise<Array<import("../types").SpatialGranularity>>}
 */
 export function getSpatialGranularities () {
-  return api.get("/spatial/granularities").then(resp => resp.data);
+  return api.get(getLocalizedUrl("/spatial/granularities/")).then(resp => resp.data);
```

Comment:
Remove first `/` to fix the 404 error.
I can't make the suggestion on github somehow.